crates/router/src/connector/iatapay/transformers.rs

```diff
@@ -473,3 +477,30 @@ pub struct IatapayAccessTokenErrorResponse {
     pub error: String,
     pub path: String,
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct IatapayWebhookResponse {
+    pub status: IatapayWebhookStatus,
+    pub iata_payment_id: Option<String>,
+    pub iata_refund_id: Option<String>,
+}
```

Comment:
Update this to an enum with 2 different struct, as handling them with options will cause us to consume wrong status and id according to our current implementation if both of them are not present